lua/neogit/lib/buffer.lua

```diff
@@ -83,6 +88,48 @@ function Buffer:set_lines(first, last, strict, lines)
   api.nvim_buf_set_lines(self.handle, first, last, strict, lines)
 end
 
+function Buffer:set_line_buffer(line)
```

Comment:
I suppose this means *buffering*/*deferred*, and not related to a vim buffer/file?

If so, perhaps `buffered_set_line` would make the distiction clearer

Reply:
Yeah, the wording is weird. I'll change it.